src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -944,6 +948,21 @@ public sealed override bool IsSerializable
             }
         }
 
+        public sealed override bool AreLocalsZeroed
+        {
+            get
+            {
+                var data = this.GetDecodedWellKnownAttributeData();
+
+                if (data != null)
+                {
+                    return !data.HasSkipLocalsInitAttribute;
```

Comment:
>return !data.HasSkipLocalsInitAttribute; [](start = 20, length = 40)

This looks wrong.

Reply:
Yes, that's wrong. Just forgot to update this. I'll fix it and add a test.